src/modules/public/markets/reducer.ts

```diff
@@ -41,10 +47,22 @@ export const marketsReducer = (state = initialMarketsState, action: MarketsActio
                 timestamp: Math.floor(Date.now() / 1000),
             };
         case MARKETS_DATA:
+            let filters = {};
+
+            if (isFinexEnabled() && action.payload) {
+                filters = action.payload.reduce((result, market: Market) => {
+                    result[market.id] = result[market.id] || [];
+                    result[market.id] = (market.filters || []).map(buildFilterPrice);
```

Comment:
are we overriding result[market.id] twice?

Reply:
got it!)